hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1180,7 +1180,7 @@
       Default replication value. The actual number of replications can be
       specified when writing the key. The default is used if replication
       is not specified when creating key or no default replication set at
-      bucket. Supported values: 1, 3 and EC_3_2.
+      bucket. Supported values: 1, 3 and RS-3-2-1024k.
```

Comment:
Hi @swamirishi, thank you for working on this. For EC we will support RS-3-2-1024K (or) RS-6-3-1024K (or) RS-10-4-1024K. We will also support XOR, so, in the place of RS, we can configure XOR. Example: XOR-3-2-1024K.

Reply:
Please update this patch as per the discussion we had offline.